03 Tests samples/src/components/MessagesSection/MessagesSection.test.tsx

```diff
@@ -27,5 +28,10 @@ describe('Message List component test', () => {
   it('should getMessages be called in ComponentDidMound', () =>{
     expect(getMessagesSpy.calledOnce).toBe(true);
   });
+  it('should handle the click event', () =>{
+    expect(
+      typeof component.find(Button).at(0).prop('onClickButton')).toBe('function');
```

Comment:
Another very useful test here is the one in which you call the function handled in the prop:
 component.find(Button).at(3).prop('handleOnClick')();
 expect(props.handleOnClick).toHaveBeenCalledTimes(1);

Reply:
Done, I had forgotten to upload the changes!